index.tsx

```
@@ -121,31 +263,170 @@ export default definePlugin({
+
+            if (!settings.store.colorNames)
+                return <>{prefix}{username}</>;
+
+            const pkAuthor = userSystem ?? getAuthorOfMessage(message, pluralKit.api);
+
+            // A PK message without an author. It's likely still loading
+            if (!pkAuthor)
+                return <span style={{color: '#555555'}}>{prefix}{username}</span>;
+
+            if (pkAuthor.switches) {
+                const [messageSwitch] = pkAuthor.switches?.values()?.filter((switchObj) => {return message.timestamp >= switchObj.timestamp});
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)